esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -202,15 +202,25 @@ define([
             }
 
             if (!this.dropZoneMachineSelect.initalized) {
+                var selectedOS;
                 this.dropZoneMachineSelect.init({
                     DropZoneMachines: true,
                     callback: function (value, row) {
+                        var path = targetRow.machine.Directory.indexOf("/");
                         targetRow.machine.Name = value
                         targetRow.machine.Netaddress = value
                         if (context.dropZoneFolderSelect) {
                             context.dropZoneFolderSelect._dropZoneTarget = targetRow;
                             context.dropZoneFolderSelect.defaultValue = "/"
-                            context.dropZoneFolderSelect.loadDropZoneFolders();
+                            if (path > -1) {
```

Comment:
Seems to be defaulting to Windows if there is no / in the path - I would have thought defaulting to linux unless there is a \ in the path was more appropriate?